cmd/generate-index/main.go

```diff
@@ -49,17 +49,29 @@ func process() error {
 	}
 	defer outputFile.Close()
 
-	_, _ = outputFile.WriteString("package cdncheck\n\nvar generatedData = &InputCompiled{\n")
+	// function to trim the last comma from an array
+	endOfArray := func(len int) func() string {
```

Comment:
Unless I'm missing something, can't we use standard json marshaler (`json.Marshal` or `json.MarshalIndent`)?